RevenueCat/Plugins/iOS/PurchasesUnityHelper.m

```diff
@@ -491,6 +495,10 @@ void _RCSetDebugLogsEnabled(const BOOL enabled) {
     [_RCUnityHelperShared() setDebugLogsEnabled:enabled];
 }
 
+void _RCSetLogLevel(const char *level) {
```

Comment:
Hmm do we need to update the android counterpart?

Reply:
Hmm update what? Did I miss something?

Reply:
Oh duh, I see what you're saying, one sec.

Reply:
Added to Android but I haven't been able to test it yet.